spec/services/workflow_errors_reporter_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe WorkflowErrorsReporter do
+  let(:put) { 'https://sul-lyberservices-test.stanford.edu/workflow/dor/objects/druid:jj925bx9565/workflows/preservationWF/moab-valid' }
+  let(:headers) { { 'Content-Type' => 'application/xml' } }
+  let(:result_array) do
+    { 13 => "Invalid moab, validation error...ential version directories." }
+  end
+  let(:body) { "<process name='moab-valid status='error' errorMessage='{13=>\"Invalid moab, validation error...ential version directories.\"}'/>" }
+
+  describe '.update_workflow with 204' do
+    before do
+      Settings.workflow_services_url = 'https://sul-lyberservices-test.stanford.edu/workflow/'
+      stub_request(:put, put)
+         .with(body: body,
+              headers: headers)
+         .to_return(status: 204, body: "", headers: {})
+    end
+
+    it 'returns a 204 response status with rails info' do
+      # used debug here because i couldn't get receive(:info) to work??
+      expect(Rails.logger).to receive(:debug).with("put request went through")
```

Comment:
i think this is debug level information, so we are ok.  I'm sure we could get 'info' to work but would rather move on.  Remove the comment of line 21?